mocked-api/schemasData.ts

```diff
@@ -66,6 +74,54 @@ export const schemaCatalogData = [
 ];
 
 export const schemasData: { [key: string]: object } = {
+  "ansible_tower_job_template_sink_0.1": {
+    type: "object",
+    additionalProperties: false,
+    properties: {
+      endpoint: {
+        type: "string",
+        title: "Endpoint",
+        description: "Ansible Tower instance base endpoint.",
+        pattern:
+          "(http|https):\\/\\/([\\w_-]+(?:(?:\\.[\\w_-]+)+))([\\w.,@?^=%&:\\/~+#-]*[\\w@?^=%&\\/~+#-])",
+        example: "https://my.ansible-tower.host",
```

Comment:
is it expected if an inappropriate value is put into `endpoint`, always just regexp is shown? I was not able to get an example of value.

Reply:
I think yes.